src/raudio.c

```diff
@@ -332,6 +333,8 @@ struct rAudioBuffer {
 
     rAudioBuffer *next;             // Next audio buffer on the list
     rAudioBuffer *prev;             // Previous audio buffer on the list
+
+    float panVolumeSmoothing[2];
```

Comment:
Could this variable be omitted in some way? If I understand it correctly it's used to make the pan transition smoother, couldn't that be controlled by users? similar to pitch change...

Reply:
I have removed smoothing entirely. This feature is now a simple API for statically positioning a sound in the stereo field.

For dynamic pan changes, you need per-sample smoothing, which at the moment, is not feasible by users, as they can only modify things for a whole audio block. However, I'm currently adding a new feature, that will allow users to add audio processing effects - and this will give them full per-sample control if they need it.